cmd/parsebatchl2data/parsebatchl2data.go

```diff
@@ -0,0 +1,128 @@
+package parsebatchl2data
+
+import (
+	_ "embed"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"strings"
+)
+
+var (
+	//go:embed usage.md
+	usage         string
+	inputFileName *string
+)
+
+var ParseBatchL2Data = &cobra.Command{
+	Use:     "parse-batch-l2-data [flags]",
+	Aliases: []string{"parsebatchl2data"},
+	Short:   "Convert batch l2 data into an ndjson stream",
+	Long:    usage,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		rawData, err := getInputData(cmd, args)
+		if err != nil {
+			return err
+		}
+		batchL2Data := strings.TrimSpace(strings.TrimPrefix(string(rawData), "0x"))
+		rawBatchL2Data, err := hex.DecodeString(batchL2Data)
+		if err != nil {
+			log.Err(err).Msg("Unable to hex decode batch l2 dat")
+			return err
+		}
+
+		rawBatch, err := DecodeBatchV2(rawBatchL2Data)
+		if err != nil {
+			log.Error().Err(err).Msg("unable to decode l2 batch data")
+			tryRawBatch(rawBatchL2Data)
+			return err
+		}
```

Comment:
nit: I don't get the difference between `rawData`, `rawBatchL2Data` and `rawBatch`, they mean the same to me :(